.github/workflows/test-on-droplets-matrix.yml

```diff
@@ -37,6 +37,32 @@ jobs:
         run: |
           sudo apt-get install libsystemd-dev cmake libdbus-1-dev libglib2.0-dev
 
+      - name: Download and build required files for running tests
+        run: |
```

Comment:
Lot of commands and magic  version numbers, there is a risk that we don't remember why this is needed or to  properly keep  this up to date, we should probably add more comments or link in the doc where these instructions come from.

Reply:
These are copied from `packaging/Makefike`